### PR TITLE
IW-1349 | Support for <panel> feature in portable infobox

### PR DESCRIPTION
**Pull request by rybmat**
## Links

* http://wikia-inc.atlassian.net/browse/IW-1349
* https://github.com/Wikia/app/pull/16482

## Reviewers

@Wikia/iwing

### CODE REVIEW

File: tasks/assets-size/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const assetMap = require('../../dist/mobile-wiki/assets/assetMap').assets;
 
 // set maximum allowed sizes for each asset in kilobytes
 const assetsSizes = {
-  'app.css': 87,
+  'app.css': 90,
```
Comment (hakubo) [inline tasks/assets-size/index.js]: please update your branch
this is now defined in
https://github.com/Wikia/mobile-wiki/blob/dev/config/bundlesize.js

Comment (rybmat) [reply tasks/assets-size/index.js]: @hakubo we'll be merging it to release branch so we'll do it later

File: app/styles/module/article/_portable-infobox.scss [hunk 2]
```diff
@@ -85,6 +85,67 @@ $infobox-box-shadow: 0 6px 10px 0 rgba($wds-color-black, .1);
 
 //////////// General classes and rules used across infobox ////////////
 
+.pi-section-navigation {
+  $overflow-color: rgba($wds-color-black, 0.25);
+
+  @extend .wds-tabs;
```
Comment (hakubo) [inline app/styles/module/article/_portable-infobox.scss]: would that mean that styles for wds-tabs will end up twice in CSS?
could you add class to pi-section-navigation so it looks/behaves like a tab?

There are separate templates for infoboxes for mobile in app.

File: app/components/article-content.js [hunk 3]
```diff
@@ -135,13 +135,34 @@ export default Component.extend(
...
       return undefined;
     },
 
+    handleInfoboxClick(event) {
+      const newActiveToggle = event.target.closest('.pi-section-tab');
+
+      if (newActiveToggle) {
+        const panel = event.target.closest('.pi-panel');
+        const newRef = newActiveToggle.getAttribute('data-ref');
+        const oldActiveToggle = panel.querySelector('.pi-section-tab.pi-section-active');
```
Comment (hakubo) [inline app/components/article-content.js]: currentActiveToggle/currentActiveContent

**Comment by rybmat** [comment]
PLEASE DO NOT RESOLVE CONFLICTS until https://github.com/Wikia/mobile-wiki/pull/1276 is merged

File: config/bundlesize.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ module.exports = {
   },
   'app.css': {
     pattern: `${assetsFolder}/app.css`,
-    limit: '87KB',
+    limit: '90KB',
```
Comment (mszabo-wikia) [inline config/bundlesize.js:16]: 🍔

**Review by mszabo-wikia — APPROVED** [review]

**Pull request merged**